command/certificate/install.go

```diff
@@ -0,0 +1,205 @@
+)
+
+func installCommand() cli.Command {
+	return cli.Command{
+		Name:   "install",
+		Action: command.ActionFunc(installAction),
+		Usage:  "install a certificate in the system truststore",
+		UsageText: `**step certificate install** <crt-file>
+		[**--prefix**=<name>] [**--all**]
+		[**--java**] [**--firefox**] [**--no-system**]`,
+		Description: `**step certificate install** installs a certificate in the system truststore.
+
+Java and Firefox truststores are also supported if the properly flags are
+passed.
+
+## POSITIONAL ARGUMENTS
+
+<crt-file>
+:  Certificate to install in the system truststore
+
+## EXAMPLES
+
+Install a certificate in the system truststore:
+'''
+$ step certificate install root-ca.pem
+'''
+
+Install a certificate in all the supported truststores:
+'''
+$ step certificate install -all root-ca.pem
+'''`,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name: "prefix",
+				Usage: `The prefix used to <name> the CA in the truststore. Defaults to the
+certificate common name.`,
+			},
+			cli.BoolFlag{
+				Name:  "java",
+				Usage: "install on the Java key store",
+			},
+			cli.BoolFlag{
+				Name:  "firefox",
+				Usage: "install on the Firefox NSS security database",
+			},
+			cli.BoolFlag{
+				Name:  "no-system",
+				Usage: "disables the install on the system truststore",
+			},
+			cli.BoolFlag{
+				Name:  "all",
+				Usage: "install on the system, Firefox and Java truststores",
+			},
+		},
+	}
+}
+
+func uninstallCommand() cli.Command {
+	return cli.Command{
+		Name:   "uninstall",
+		Action: command.ActionFunc(uninstallAction),
+		Usage:  "uninstall a certificate from the system truststore",
+		UsageText: `**step certificate uninstall** <crt-file>
+		[**--prefix**=<name>] [**--all**]
+		[**--java**] [**--firefox**] [**--no-system**]`,
+		Description: `**step certificate install** uninstalls a certificate from the system truststore.
+
+Java and Firefox truststores are also supported if the properly flags are
+passed.
+
+## POSITIONAL ARGUMENTS
+
+<crt-file>
+:  Certificate to uninstall from the system truststore
+
+## EXAMPLES
+
+Uninstall from only the system truststore:
+'''
+$ step certificate uninstall root-ca.pem
+'''
+
+Uninstall a certificate from all the supported truststores:
+'''
+$ step certificate uninstall -all root-ca.pem
+'''`,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name: "prefix",
+				Usage: `The prefix used to <name> the CA in the truststore. Defaults to the
+certificate common name.`,
+			},
+			cli.BoolFlag{
+				Name:  "java",
+				Usage: "uninstall from the Java key store",
+			},
+			cli.BoolFlag{
+				Name:  "firefox",
+				Usage: "uninstall from the Firefox NSS security database",
+			},
+			cli.BoolFlag{
+				Name:  "no-system",
+				Usage: "disables the uninstall from the system truststore",
+			},
+			cli.BoolFlag{
+				Name:  "all",
+				Usage: "uninstall from the system, Firefox and Java truststores",
+			},
+		},
+	}
+}
+
+func installAction(ctx *cli.Context) error {
```

Comment:
maybe check that the certificate is a root certificate. otherwise the error returned from `truststore` is pretty opaque.